pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -210,21 +214,60 @@ public String listTableConfigs(@ApiParam(value = "realtime|offline") @QueryParam
 
       if (tableType == null) {
         tableNames = _pinotHelixResourceManager.getAllRawTables();
+      } else if (tableType == TableType.REALTIME) {
+        tableNames = _pinotHelixResourceManager.getAllRealtimeTables();
       } else {
-        if (tableType == TableType.REALTIME) {
-          tableNames = _pinotHelixResourceManager.getAllRealtimeTables();
-        } else {
-          tableNames = _pinotHelixResourceManager.getAllOfflineTables();
-        }
+        tableNames = _pinotHelixResourceManager.getAllOfflineTables();
+      }
+      if (sortType == null) {
```

Comment:
We should treat this the same as `sortType` of "name" for backward-compatibility
```suggestion
      if (sortType == null || sortType.equalsIgnoreCase("name")) {
        ...
      }
```

Reply:
Sure